<launch>
	<arg name="file_name" default="/known_locs_apartment.yaml"/>

	<node name="known_locs" pkg="known_locations_tf_server" type="setbool_service_locs_tf.py" output="screen">
		
		<param name="file_name" value="$(arg file_name)"/>

	</node>


</launch>